pallets/phala/src/stakepool.rs

```diff
@@ -47,6 +47,7 @@ pub mod pallet {
 
 	const STAKING_ID: LockIdentifier = *b"phala/sp";
 
+	const MAX_WHITELIST_LEN: u32 = 500;
```

Comment:
One empty line after this line. `const` doesn't belong to `Ledger`'s scope.

Reply:
change to 100